src/explore.sh

```diff
@@ -40,15 +44,19 @@ function explore()
 function explore_parser()
 {
   if [[ "$#" -eq 0 ]]; then
-    complain "Expected string or 'log'"
+    complain "Expected string or 'log' or 'grep'"
     exit 22 # EINVAL
   fi
 
   if [[ "$1" =~ ^--log$ && ! -z "$2" ]]; then
     return 1
   fi
+  
+  if [[ "$1" =~ ^--grep$ && ! -z "$2" ]]; then
```

Comment:
in this case, it could be nice if we also support `-g`, i.e., `kw e -g lala` or `kw e --grep lala`.